### ISSUE TITLE
var(): argument 'dim' must be int, not list

### ISSUE THREAD

**Issue by dxm343488059** [znxlwm/UGATIT-pytorch#38]
`torch.mean(input, dim=[2, 3], keepdim=True), torch.var(input, dim=[2, 3], keepdim=True)`

Modify the code so that it resembles the following code sample:

`torch.mean(torch.mean(input, dim=2, keepdim=True), dim=3, keepdim=True), torch.var(torch.var(input, dim=2, keepdim=True), dim=3, keepdim=True)`

**Comment by AstomMars** [comment]
It works, thanks a lot. The problem seems to be that version of pytorch is too old.
